lemur/static/app/angular/api_keys/services.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+angular.module('lemur')
+  .service('ApiKeyApi', function (LemurRestangular) {
+    return LemurRestangular.all('api_keys');
+  })
+  .service('ApiKeyService', function ($location, ApiKeyApi) {
+    var ApiKeyService = this;
+    ApiKeyService.update = function(apiKey) {
+      return apiKey.put();
+    };
+
+    ApiKeyService.updateRevoked = function(apiKey) {
```

Comment:
Is this needed? Is `update` enough?

Reply:
Probably, to be honestI'm not sure why I added two here.